lib/kdl/include/kdl/vector_utils.h

```diff
@@ -224,26 +247,48 @@ namespace kdl {
      * @param args the vectors to append to v
      */
     template<typename T, typename A, typename... Args>
-    void vec_append(std::vector<T, A>& v, const Args& ... args) {
+    void vec_append(std::vector<T, A>& v, Args&&... args) {
         v.reserve(kdl::col_total_size(v, args...));
-        (..., v.insert(std::end(v), std::begin(args), std::end(args)));
+        detail::vec_append(v, std::forward<Args>(args)...);
     }
 
     /**
      * Returns a vector that contains all elements from the given vectors v and the elements of args in the order in
```

Comment:
`given vectors v` should be changed to `given vector first`

Reply:
Fixed 0d1af3bdb.